client/ayon_equalizer/plugins/load/load_plate.py

```diff
@@ -43,84 +47,99 @@ class LoadPlate(load.LoaderPlugin):
     icon = "code-fork"
     color = "orange"
 
-    def load(self, context, name=None, namespace=None, options=None):
-        representation = context["representation"]
-        project_name = get_current_project_name()
-        version = get_version_by_id(project_name, representation["parent"])
+    def load(self, context: dict, name: Optional[str] = None,
+             namespace: Optional[str] = None, options: Optional[dict]=None):
+        """Load image sequence to the current camera."""
+        repre_entity = context["representation"]
+        version_entity = context["version"]
+        version_attributes = version_entity["attrib"]
 
-        file_path = self.file_path(representation, context)
+        file_path = get_representation_path(repre_entity)
```

Comment:
You can use implemented method on `LoaderPlugin`.
```suggestion
        file_path = self.filepath_from_context(repre_entity)
```